control/init.sh

```diff
@@ -60,11 +60,22 @@ if [ -d ${COMMCARE_CLOUD_REPO}/commcare-cloud ]; then
     rm -rf ${COMMCARE_CLOUD_REPO}/commcare-cloud
 fi
 
+function uninstall-lowerversion-ansible() {
+    ANSIBLE_VERSION=`pip show ansible | grep Version | awk '{print $2}'`
+    if [[ ${ANSIBLE_VERSION:0:3} < "4.0" ]] && [[ ! -z ${ANSIBLE_VERSION} ]]; then
+        echo "installed version of ansible is: ${ANSIBLE_VERSION:0:3}"
+        echo "ansible version ${ANSIBLE_VERSION} is uninstalling"
+        pip uninstall ansible --yes
+    fi
+}
```

Comment:
This should work great for updating commcare-cloud on the control machine, but it won't help people with their own dev environment. I suggest we include a similar check in commcare-cloud that tells people to upgrade ansible. (It doesn't have to block them, just alert people that to upgrade they have to first run this uninstall step.)